packages/manager/src/features/Volumes/VolumeCreate.tsx

```diff
@@ -358,7 +358,7 @@ export const VolumeCreate = () => {
           variant="error"
         />
       )}
-      <form onSubmit={handleSubmit}>
+      <form noValidate onSubmit={handleSubmit}>
```

Comment:
```suggestion
      <form onSubmit={handleSubmit}>
```

This may no longer be needed